utilities/tools/helper/Get-PipelineFileName.ps1

```diff
@@ -0,0 +1,38 @@
+﻿<#
+.SYNOPSIS
+Find the correct yml pipeline naming for a given resource identifier.
+
+.DESCRIPTION
+Find the correct yml pipeline naming for a given resource identifier.
+If a child resource type is provided, the corresponding yml pipeline name is the one of its parent resource type
+
+.PARAMETER ResourceIdentifier
+Mandatory. The resource identifier to search for, i.e. the relative module file path starting from the resource provider folder.
+
+.EXAMPLE
+Get-PipelineFileName -ResourceIdentifier 'storage/storage-account/blob-service/container/immutability-policy'.
+
+Returns 'ms.storage.storageaccounts.yml'.
+
+.EXAMPLE
+Get-PipelineFileName -ResourceIdentifier 'storage/storage-account'.
+
+Returns 'ms.storage.storageaccounts.yml'.
+#>
+function Get-PipelineFileName {
+
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true)]
+        [string] $ResourceIdentifier
+    )
+
+    . (Join-Path $PSScriptRoot 'Get-SpecsAlignedResourceName.ps1')
+
+    $provider, $parentType, $childTypeString = $ResourceIdentifier -split '/', 3
```

Comment:
```suggestion
    $provider, $parentType, $childTypeString = $ResourceIdentifier -split '[\/||\\]', 3
```
Can it not also be the other way around?

Reply:
Meaning either '/' or '\'? If called locally maybe. Would you then suggest to replace to a common approach?

Reply:
I think what we usually have is this `replace('\\','/').split('/')` and just recently for the scripts I proposed for the folder renaming stuff I used support for both using the `['\\|\/]`. I don't think we need to change this anywhere else as we don't have a consistent use of `split() | -split` or `replace() | -replace` anyways.

Long story short, I would suggest to use it here - be it only for the sake of robustness.